src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -896,6 +896,31 @@ public ValidateLengthAttribute(int minLength, int maxLength) : base()
         }
     }
 
+    /// <Summary>
+    /// Predefined range kind to use with ValidateRangeAttribute.
+    /// </Summary>
+    public enum ValidateRangeKind {
```

Comment:
Please follow the conventions of the file you are editing, e.g. braces normally go on their own line, and you have no space after `if`.